homeassistant/components/twinkly/light.py

```diff
@@ -193,7 +195,7 @@ async def async_turn_on(self, **kwargs: Any) -> None:
 
             await self._client.set_brightness(brightness)
 
-        if (
+        elif (
```

Comment:
Could not potentially both color and brightness be set in the same turn_on() call?